.azure/modules/keyvault/copySecrets.bicep

```diff
@@ -9,28 +9,45 @@ param destKeyVaultName string
 param destKeyVaultRGName string = resourceGroup().name
 param destKeyVaultSubId string = subscription().subscriptionId
 
+// App configuration
+param appConfigurationName string
+
 // Secret
 #disable-next-line secure-secrets-in-params
 param secretPrefix string
-param removeSecretPrefix bool = true
 
 var environmentKeys = [for key in srcKeyVaultKeys: {
     isEnvironmentKey: startsWith(key, secretPrefix)
-    value: removeSecretPrefix ? replace(key, secretPrefix, '') : key
-    fullName: key
+    secretName: key
+    appConfigKey: replace(replace(key, secretPrefix, ''), '--', ':')
 }]
 
 resource srcKeyVaultResource 'Microsoft.KeyVault/vaults@2023-07-01' existing = {
 	name: srcKeyVaultName
     scope: resourceGroup(srcKeyVaultSubId, srcKeyVaultRGNName)
 }
 
+resource appConfigurationResource 'Microsoft.AppConfiguration/configurationStores@2023-03-01' existing = {
+    name: appConfigurationName
+}
+
 module secrets 'upsertSecret.bicep' = [for key in environmentKeys: if (key.isEnvironmentKey) {
-    name: '${take(key.value, 57)}-${take(uniqueString(key.value), 6)}'
+    name: '${take(key.secretName, 57)}-${take(uniqueString(key.secretName), 6)}'
     scope: resourceGroup(destKeyVaultSubId, destKeyVaultRGName)
     params: {
         destKeyVaultName: destKeyVaultName
-        secretName: key.value
-        secretValue: srcKeyVaultResource.getSecret(key.fullName)
+        secretName: key.secretName
```

Comment:
Dette er feil? Her får du med secret prefix. Si at du har en any secret og en env spesifikk secret. Hvordan skal env overskrive any secreten hvis de har forskjellige navn? 

Vent litt - dette virker å være gjort med vilje, og at du i appconfig gjør mappinga til den faktiske secreten? Så hvis vi har 
- any-foo
- test-foo

Så vil begge være i test KV, men app config vil kun inneholde foo som peker på test-foo? Fint for å se hvor hemmelighetene har kommet fra, men alle any secrets som har en env spesifikk en vil være useless. Gjort med vilje?